en/docs/Learn/APIMonetization/monetizing-an-api.md

```diff
@@ -0,0 +1,703 @@
+# Monetizing an API
```

Comment:
Overall,  It is better if we can tell what to do in configuration files in one go, rather than having to stop the server in the middle.  Else it will be the same problem we had with the upgrading guide.

Reply:
Yes, I totally agree. Irrespective of the initially draft when I put the MD docs together I wrote the documentation with the config all together. However Chamin advised otherwise. However, after your comment I checked with Uvindra and he too was on the same page as you. So I have added all the config together now.